src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_rpc_plugin.cc

```diff
@@ -91,9 +91,7 @@ void RCRPCPlugin::OnApplicationEvent(
   switch (event) {
     case plugins::kApplicationRegistered: {
       application->AddExtension(new RCAppExtension(kRCPluginID));
-      if (resource_allocation_manager_->is_rc_enabled()) {
-        resource_allocation_manager_->SendOnRCStatusNotification();
-      }
+      resource_allocation_manager_->SendOnRCStatusNotification(true);
```

Comment:
@LitvinenkoIra are we adding parameter that RC is allowed or not only on application registration?

Reply:
@LitvinenkoIra where do we add this param on enabling/disabling RC?

Reply:
@LuxoftAKutsan when we enabling/disabling RC we should send OnRCStatus notification only to mobile and we do it here
https://github.com/LitvinenkoIra/sdl_core/blob/feature/onRcStatus_allowed/src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/resource_allocation_manager_impl.cc#L334